toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -192,6 +195,15 @@ func CustomizeImageWithConfigFile(buildDir string, configFile string, inputImage
 	var err error
 
 	var config imagecustomizerapi.Config
+
+	ctx := context.Background()
+	_, span := Tracer.Start(ctx, "customizeImage")
```

Comment:
`CustomizeImage` is probably the more interesting function to trace. (Though you can trace both if you really want to include parsing the YAML file.)